test/Serialization/Address.purs

```diff
@@ -48,117 +64,95 @@ mPkh = ed25519KeyHashFromBech32 pkhBech32
 mScriptHash :: Maybe ScriptHash
 mScriptHash = scriptHashFromBytes $ hexToByteArrayUnsafe scriptHashHex
 
-suite :: TestPlanM Unit
-suite = do
+addressFunctionsTest :: TestPlanM Unit
```

Comment:
Why don't you use `test` and `group`? This suite will exit on first fail, I find this not very convenient. Plus, it not possible to use `only` and `skip` helpers to focus on some part of tests when developing.